scripts/pi-hole/php/groups.php

```diff
@@ -920,7 +920,11 @@ function JSON_error($message = null)
                 continue;
             }
 
-            if(preg_match("/[^a-zA-Z0-9:\/?&%=~._()-;]/", $address) !== 0) {
+            // this will remove first @ that is after schema and before domain
+          ￼ // $1 is optional schema, $2 is userinfo
```

Comment:
```suggestion
           // $1 is optional schema, $2 is userinfo
```

Not sure where this char has come from, but verified it works without it there!